tests/unit/auth-services.spec.ts

```diff
@@ -48,7 +48,7 @@ describe('Auth Services', () => {
     const userInfo = await AuthServices.fetchUserInfo()
 
     // verify data
-    expect(userInfo).toEqual(USER_INFO)
+    expect(userInfo).toEqual({ data: USER_INFO })
```

Comment:
Needed because `fetchUserInfo()` now returns axios response, not user info object.